<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resource/image/softIcon.ico</normaloff>:/resource/image/softIcon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QWidget" name="widget_para" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>选择路径：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_selectedFile">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>5</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>打开</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/resource/image/open.ico</normaloff>:/resource/image/open.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>检测类型：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_detType">
              <item>
               <property name="text">
                <string>图片（单张）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>图片（文件夹）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>视频（本地）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>视频（相机）</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>模型：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_model">
              <item>
               <property name="text">
                <string>yolov8s</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>yolov8m</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>yolov8x</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="minimumSize">
            <size>
             <width>5</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>启动</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/resource/image/play.ico</normaloff>:/resource/image/play.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_end">
           <property name="text">
            <string>停止</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/resource/image/stop.ico</normaloff>:/resource/image/stop.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_show" native="true">
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="7,3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pic">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
